### PR TITLE
✨ Render next figure

### PR DESCRIPTION
**Pull request by alena-bartosh**
Screenshot from 2022-11-11 14-24-03

### CODE REVIEW

File: src/renderer.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 from board import Board
+from figure import Figure
+from point import Point
 from score_holder import ScoreHolder
 
 
 class Renderer:
     __BORDER_WEIGHT: int = 1
-    __STATISTIC_TOP_PADDING: int = 1
+    __FIGURE_TOP_PADDING: int = 1
```
Comment (s373r) [inline src/renderer.py]: __NEXT_FIGURE_TOP_PADDING

File: src/renderer.py [hunk 2]
```diff
@@ -44,13 +47,28 @@ def __draw_board_cells(self, board: Board) -> None:
                 self.__draw_cell(x, y, is_cell_fill)
 
     # TODO(DP): use Figure class instead of figure_points
-    def __draw_figure(self, figure_points: list) -> None:
+    def __draw_figure(self, figure_points: list[Point]) -> None:
         for point in figure_points:
             self.__draw_cell(point.x, point.y, is_fill=True)
 
+    def __get_right_board_point(self, board: Board) -> int:
```
Comment (s373r) [inline src/renderer.py]: __get_y_outside_board()

**Review by s373r — CHANGES_REQUESTED** [review]
Added 2 minor comment

File: src/renderer.py (hunk 1 above)
Comment (alena-bartosh) [reply src/renderer.py]: Done

File: src/renderer.py (hunk 2 above)
Comment (alena-bartosh) [reply src/renderer.py]: Done

**Review by Serg-Bartosh — APPROVED** [review]

**Review by s373r — APPROVED** [review]

**Pull request merged**